src/chainlink/VRFSubscription.ts

```diff
@@ -0,0 +1,106 @@
+import { writeContract } from "../core/helpers/util";
+
+
+export class VRFSubscription {
```

Comment:
move VRFSubscription & VRFCoordinator to a "vrf" subfolder: 
`./chainlink/vrf/VRFCoordinator.ts`
`./chainlink/vrf/VRFSubscription.ts`